go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/gin-gonic/gin v1.5.0
 	github.com/radovskyb/watcher v1.0.7
+	github.com/roboticeyes/gorexos v0.0.0-20200605091724-576326b6cdd1
```

Comment:
dependency to gorexos is not allowed by design!

Reply:
removed